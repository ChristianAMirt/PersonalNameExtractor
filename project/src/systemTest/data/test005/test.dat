<NER>On the year of 2024, John Ventured out to Mississippi. There he was able to meet new friends. The first friend he met was Sarah Jacobson. Sarah was an athlete from New York. The second friend was Michael Smith. Michael, who usually like to go by Mike,  was a Doctor who worked at the Memorial Herman Hospital in Houston, Texas. John is enjoying life in Mississippi.</NER>