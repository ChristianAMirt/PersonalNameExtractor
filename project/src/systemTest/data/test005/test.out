<NER>On the year of 2024 , <PER> John Ventured out to </PER> Mississippi . There he </PER> was able </PER> to </PER> meet new </PER> friends </PER> . The first </PER> friend </PER> he </PER> met was <PER> Sarah Jacobson </PER> . <PER> Sarah was <PER> an athlete from </PER> New </PER> York </PER> . The second </PER> friend </PER> was <PER> Michael Smith </PER> . <PER> Michael , who usually like </PER> to </PER> go </PER> by <PER> Mike ,  was a Doctor </PER> who worked at the Memorial <PER> Herman Hospital <PER> in <PER> Houston , Texas . <PER> John is enjoying life </PER> <PER> in Mississippi .</NER> 