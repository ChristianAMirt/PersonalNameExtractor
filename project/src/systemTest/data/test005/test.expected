<NER>On the year of 2024, <PER>John<\PER> ventured out to Mississippi. There he was able to meet new friends. The first friend he met was <PER>Sarah Jacobson<\PER>. <PER>Sarah<\PER> was an athlete from New York. The second friend was <PER>Michael Smith<\PER>. <PER>Michael<\PER>, who usually like to go by <PER>Mike<\PER>,  was a Doctor who worked at the Memorial Herman Hospital in Houston, Texas. <PER>John<\PER> is enjoying life in Mississippi.</NER>